### ISSUE TITLE
Form with tabs

### ISSUE THREAD

**Issue by cerelli** [crocodic-studio/crudbooster#809]
Hi, i want to reproduce with crudbooster a form like this in image:
![00crm](https://user-images.githubusercontent.com/6558535/30164633-6ecdc738-93dd-11e7-906b-d6e8d64b4971.png)
In form, add tabs with details subform. In subform panel and button to modify or delete.

is it possible with crudbooster ?

**Comment by DeveloperOnCall** [comment]
Yes. This is possible by replacing the getAdd() method of the generated AdminXXXXController class.

Here is some more reading on this.
[Add Custom Add Form](https://github.com/crocodic-studio/crudbooster/blob/master/docs/en/how-custom-view-add.md)

Hope that helps.

**Comment by cerelli** [comment]
crudbooster does not use eloquent, right? Can there be no performance issues? If I have a form with so many tabs, do I have to load them all at once?
